Header header
HDMap/msg_road curr_road
HDMap/msg_junction curr_junc
HDMap/msg_road next_road
